packages/create-mud/src/cli.ts

```diff
@@ -16,7 +16,8 @@ create("create-mud", {
   promptForEmail: false,
   promptForLicense: false,
   promptForTemplate: true,
-  caveat: ({ answers }) => `Done! Play in the MUD with \`cd ${answers.name}\` and \`yarn dev\``,
+  caveat: ({ answers, packageManager }) =>
+    `Done! Play in the MUD with \`cd ${answers.name}\` and \`${packageManager} run dev\``,
```

Comment:
We still require folks to use pnpm in the templates right? Should we add a note about that here?

Reply:
probably, I think I can also force the package manager here too